setup.cfg

```diff
@@ -1,2 +1,7 @@
 [flake8]
 ignore=E501
+
+[pytest]
+addopts=-r=fEsxXw --driver=Firefox
```

Comment:
We use `PYTEST_ADDOPTS` in Jenkins, and I'm not sure if this will play well with it. Also, did you know that pytest 2.8 supports `-r a` (for all).